dask/highlevelgraph.py

```diff
@@ -870,6 +870,51 @@ def merge(cls, *graphs):
         return cls(layers, dependencies)
 
     def visualize(self, filename="dask-hlg.svg", format=None, **kwargs):
+        """
+        Visualize several dask high level graphs at once.
+
+        Requires ``graphviz`` to be installed. All options that are not the dask
+        graph(s) should be passed as keyword arguments.
+
+        Parameters
+        ----------
+        dsk : dict(s) or collection(s)
```

Comment:
I just noticed that this is not actually in the arguments list (because it's going to act on self, the graph we are referring to). The text above also mentions passing graphs - this should be removed, seems like it was copied from elsewhere.

Reply:
Correct ✅
will change